deepchecks/tabular/context.py

```
@@ -102,7 +208,23 @@ def __init__(self,
         if features_importance is not None:
             if not isinstance(features_importance, pd.Series):
                 raise DeepchecksValueError('features_importance must be a pandas Series')
-
+        # TODO: a little trickery
+        if model is not None and hasattr(model, 'predict_proba'):
+            if train is not None:
+                y_proba_train = model.predict_proba(train.features_columns)
+            if test is not None:
+                y_proba_test = model.predict_proba(test.features_columns)
+        if model is not None and hasattr(model, 'predict'):
+            if train is not None:
+                y_pred_train = model.predict(train.features_columns)
+            if test is not None:
+                y_pred_test = model.predict(test.features_columns)
+        model = None
+        # we do it after validation because the model we are creating isn't valid (we are always valid just not here)
+        if model is None and \
+            not pd.Series([y_pred_train, y_pred_test, y_proba_train, y_proba_test]).isna().all():
```

do we want to raise an exception here? if one of the arrays contains na?